src/main/java/forestry/core/config/Config.java

```diff
@@ -86,6 +86,8 @@
 	public static boolean logHivePlacement = false;
 	public static boolean enableVillagers = true;
 	public static boolean generateTrees = true;
+	public static boolean replaceVanillaTrees = false;
```

Comment:
This option is unused here, probably included by mistake

Reply:
Not yet used, but put in place for #2039